swagger.yaml

```diff
@@ -2359,6 +2359,44 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /pxe-artifacts:
```

Comment:
because it's not related directly to the a specific cluster/image maybe it should be a different component and not under the installer? (inventory.go)

Reply:
Is it worth it for one method?  Let me know.

Reply:
Inventory is bit enough as it is. Do you think that more content will be added to this feature?

Reply:
I think it will only be this one API